Packs/NozomiNetworks/Integrations/NozomiNetworks/NozomiNetworks.py

```diff
@@ -6,25 +6,46 @@
 
 import urllib3
 import json
+import requests
 
 urllib3.disable_warnings()
 
 
-class Client(BaseClient):
-    def http_request(self, method, path, data=None):
-        response = self._http_request(
-            method,
-            url_suffix=path,
-            resp_type="json",
-            json_data=data
+class Client:
+    def __init__(self, base_url=None, headers=None, verify=None, proxies=None, auth=None):
+        self.base_url = base_url or demisto.params().get('endpoint')
+        self.headers = headers or {'accept': "application/json"}
+        self.verify = verify if verify is not None else not demisto.params().get('insecure', True)
+        self.proxies = proxies if proxies is not None else demisto.params().get('proxy', False)
+        self.auth = auth or (
+            demisto.params().get("credentials", {}).get('identifier', ''),
+            demisto.params().get("credentials", {}).get('password', '')
         )
-        return response
+
+    def _make_request(self, method, path, **kwargs):
+        url = self.base_url + path
+        demisto.info("Nozomi url " + url)
+        response = requests.request(
```

Comment:
It is not recommended to use requests.request,
It is safer to use _http_request from CommonServerPython.
To determine the status code they are ok You can send in the status_code parameter.